applications/colonoscopy_segmentation/colonoscopy_segmentation.py

```diff
@@ -120,7 +142,8 @@ def compose(self):
 
         is_aja = self.source.lower() == "aja"
         if is_aja:
-            source = AJASourceOp(self, name="aja", **self.kwargs("aja"))
+            aja_source = lazy_import("holohub.aja_source")
```

Comment:
Since it is a lazy import, I think placing it at the top the file makes more sense. Otherwise, we could use `import` here instead, right?

Reply:
Oh I see - let me check

Reply:
I removed the lazy import